### PR TITLE
fixes #249 - fixed svg page tempate

### LINKED ISSUE #249: Fix svg page template
Presently the svg page template (8BadgesOnA3) is not proper as it cannot be opened and shows error on opening. This is because the root node of the svg contains improper default width and height parameters.

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #249, fixes svg page template file, now it can be opened
without any error. Also the configuration of badge pages code has been
moved to a separate method and code has been refactored.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by djmgit** [comment]
@niccokunzmann @jajodiaraghav I have refactored the code and moved page configuration part to a separate method. Later we van modify this method to apply further configurations from custom json file like custom page size, badge size etc.
Also I have fixed the property of svg page template.

**Comment by vaibhavsingh97** [comment]
@djmgit PLease fix all the violations 😄

File: app/generate-badges.py [hunk 1]
```diff
@@ -21,14 +23,22 @@
     CONTENT = f.read()
 
 
-def generate_badges(aggregate, folder, index, picture, paper_size):
+def configure_badge_page(badge_page, paper_size):
     paper_width = paper_sizes[paper_size][0]
     paper_height = paper_sizes[paper_size][1]
+
```
Comment (vaibhavsingh97) [inline app/generate-badges.py]: Remove blank space

**Comment by djmgit** [comment]
@vaibhavsingh97 done !

**Pull request merged**